### PR TITLE
Fix some typo of Tide middleware evolution post

### PR DESCRIPTION
**Pull request by tirr-c**

### CODE REVIEW

File: _posts/2018-11-27-tide-middleware-evolution.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ pub struct RequestContext<'a, Data> {
 
 impl<'a, Data: Clone + Send> RequestContext<'a, Data> {
     /// Consume this context, and run remaining middleware chain to completion.
-    pub fn next(mut self) -> FutureObj<'a, Response>;
+    pub fn next(self) -> FutureObj<'a, Response> { ... }
```
Comment (tirr-c) [inline _posts/2018-11-27-tide-middleware-evolution.md]: I tried to follow the way rustdoc shows us. Not exactly same, of course!

I thought `mut` is unnecessary as we're omitting the method body here.

File: _posts/2018-11-27-tide-middleware-evolution.md [hunk 2]
```diff
@@ -240,4 +240,4 @@ Finally, a shout out to the 19 people (!) who have already contributed to Tide:
 - Yoshua Wuyts
 - csmoe
 - ibaryshnikov
-- lixiaohui
\ No newline at end of file
+- lixiaohui
```
Comment (tirr-c) [inline _posts/2018-11-27-tide-middleware-evolution.md]: I have no idea why this is considered as diff (I'm editing the post on GitHub directly).

Comment (Nemo157) [reply _posts/2018-11-27-tide-middleware-evolution.md]: Probably adding a trailing newline, normally there's a symbol displayed on the line to highlight that though 😕

**Comment by Nemo157** [comment]
Thanks 😄

File: _posts/2018-11-27-tide-middleware-evolution.md (hunk 2 above)
Comment (Nemo157) [reply _posts/2018-11-27-tide-middleware-evolution.md]: Yep, checked locally and that's what it is, checking the page source the symbol is there but set to `display: hidden` for some reason.

**Pull request merged**